include/libretro.h

```diff
@@ -528,18 +527,43 @@ enum retro_mod
                                             * If called, it should be called in retro_load_game().
                                             */
 #define RETRO_ENVIRONMENT_GET_SYSTEM_DIRECTORY 9
-                                           /* const char ** --
-                                            * Returns the "system" directory of the frontend.
-                                            * This directory can be used to store system specific
-                                            * content such as BIOSes, configuration data, etc.
-                                            * The returned value can be NULL.
-                                            * If so, no such directory is defined,
-                                            * and it's up to the implementation to find a suitable directory.
+                                           /* 
+                                            * RETURNS  ----------------------
```

Comment:
Unsure about this doc comment format for the comment titles. The rest of the file uses....

```
/* returntype --
```

We should likely stick with it, and create a new issue/pull request to handle switching it to a standard doc comment format across libretro.h entirely.

Reply:
agreed @RobLoach I have made that change, removing the "RETURNS"